sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/math.scala

```diff
@@ -254,3 +254,52 @@ case class Pow(left: Expression, right: Expression)
       """
   }
 }
+
+object Logarithm {
```

Comment:
Because we want to support the usage of `df.selectExpr("log(a)", "log(2.0, a)", "log(b)")`.